### PR TITLE
README: add language around base and optional layers

### LINKED ISSUE #120: [ReadMe] Base layer term seems overloaded
In ReadMe.md, there are references to "base layer" but seems to be an overloaded use of the term:
> The high level components of the spec include:
> •An image manifest and filesystem serialization (base layer)
> •A process of hashing the image format for integrity and content-addressing (base layer)
> •Signatures that are based on signing image content address (optional layer)
> •Naming that is federated based on DNS and can be delegated (optional layer)

Maybe change: 
"base layer" --> REQUIRED
"optional layer" --> OPTIONAL

### PR DESCRIPTION
**Pull request by philips**
Based on feedback new users might not know what the base and optional
layer language is about. Point to the scope doc that uses this language.
Fixes #120.

Signed-off-by: Brandon Philips <brandon.philips@coreos.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The high level components of the spec include:
 * Signatures that are based on signing image content address (optional layer)
 * Naming that is federated based on DNS and can be delegated (optional layer)
 
+The optional and base layers of all OCI projects are tracked in the OCI Scope Table](https://www.opencontainers.org/governance/oci-scope-table).
```
Comment (wking) [inline README.md]: “OCI” → “[OCI”

**Comment by philips** [comment]
LGTM

**Comment by philips** [comment]
@opencontainers/image-spec-maintainers any other LGTMs for this trivial change?

**Comment by stevvooe** [comment]
LGTM

**Comment by stevvooe** [comment]
@philips Does this tool allow you to LGTM your self?

**Comment by philips** [comment]
@stevvooe yes, if you are a maintainer you can LGTM yourself.

**Comment by philips** [comment]
@stevvooe but the threshold is two maintainers to merge so need one more.

**Comment by wking** [comment]
On Wed, Jun 08, 2016 at 05:47:08PM -0700, Brandon Philips wrote:
> @stevvooe yes, if you are a maintainer you can LGTM yourself.

Currently only in this repo of the OCI repos using PullApprove
(opencontainers/project-template#13, opencontainers/runtime-spec#474,
opencontainers/runc#863).

**Pull request merged**